### ISSUE TITLE
Error when installing ingress?

### ISSUE THREAD

**Issue by mikegehard** [vmware-tanzu/community-edition#2137]
**On Page:** [](/docs/latest/solutions-secure-ingress/)

When trying to install the ingress controller as part of step 3, I get the following:

```
➜  devTools tanzu package available get contour.community.tanzu.vmware.com
- Retrieving package details for contour.community.tanzu.vmware.com...
package 'contour.community.tanzu.vmware.com' does not exist in namespace 'default'
```

This is different than is what is in the docs. I thought I followed the instructions exactly.

**Comment by jpmcb** [comment]
Hi Mike - thanks for opening this issue

Have you installed the default TCE package repository?

```
$ tanzu package repository add tce-repo \ 
  --url projects.registry.vmware.com/tce/main:0.9.1 \
  --namespace tanzu-package-repo-global
```

This is noted in the "Getting started guide" [under the "Installing a package" section.](https://tanzucommunityedition.io/docs/latest/getting-started-standalone/#installing-a-package) If you're following the ["Secure automated Ingress on AWS" guide](https://tanzucommunityedition.io/docs/latest/solutions-secure-ingress/), then that is a per-requisite (and we should call that out as such in the docs)

If so, what is the output of
```
$ tanzu package repository list -A
```
and
```
$ tanzu package available list 
```
?

**Comment by mikegehard** [comment]
Ah that's the difference. I wasn't using the Getting Started guide. 

I went right to the Installation guide for AWS: https://tanzucommunityedition.io/docs/latest/aws-intro/

I'm wondering if there is a need for a bit better "guide" to getting up and running with your first standalone cluster?

The documentation is all there...it just isn't super clear, without a bunch of reading, where to get started.

I didn't read the "getting started" section because my eyes were drawn to "installation". I assumed I needed to do that first and that the getting started wouldn't cover that.
